Justfile

```diff
@@ -87,9 +87,15 @@ publish-extension: package-extension
   @cd extensions/vscode && vsce publish
 
 # Build the Kotlin runtime (if necessary)
+# build-kt-runtime:
+#  @mk {{KT_RUNTIME_OUT}} : kotlin-runtime/ftl-runtime -- mvn -f kotlin-runtime/ftl-runtime -Dmaven.test.skip=true -B install
+#  @mk {{KT_RUNTIME_RUNNER_TEMPLATE_OUT}} : {{KT_RUNTIME_OUT}} -- "mkdir -p $(dirname {{KT_RUNTIME_RUNNER_TEMPLATE_OUT}}) && install -m 0600 {{KT_RUNTIME_OUT}} {{KT_RUNTIME_RUNNER_TEMPLATE_OUT}}"
+#  @mk {{RUNNER_TEMPLATE_ZIP}} : {{KT_RUNTIME_RUNNER_TEMPLATE_OUT}} -- "cd build/template && zip -q --symlinks -r ../../{{RUNNER_TEMPLATE_ZIP}} ."
+
+# Kotlin runtime is temporarily disabled; these instructions create a dummy zip in place of the kotlin runtime jar for
+# the runner.
 build-kt-runtime:
-  @mk {{KT_RUNTIME_OUT}} : kotlin-runtime/ftl-runtime -- mvn -f kotlin-runtime/ftl-runtime -Dmaven.test.skip=true -B install
-  @mk {{KT_RUNTIME_RUNNER_TEMPLATE_OUT}} : {{KT_RUNTIME_OUT}} -- "mkdir -p $(dirname {{KT_RUNTIME_RUNNER_TEMPLATE_OUT}}) && install -m 0600 {{KT_RUNTIME_OUT}} {{KT_RUNTIME_RUNNER_TEMPLATE_OUT}}"
+  @mkdir -p build/template/ftl && touch build/template/ftl/temp.txt
```

Comment:
cc @alecthomas wdyt? this preserves the mechanism currently in place for deploying runtime-specific data to the runner— we expect an archive formerly just containing the kotlin JAR. now we have now no JAR, so we'll just archive and deploy this empty file? 🤷‍♀️

Reply:
@worstell do you want to comment out these lines as well as you did above for the `build-kt-runtime`? I assume we'll be turning this all back on at some point. Or we can delete and use git for both. whatcha think?

Reply:
Good idea, works for me.

Reply:
maybe i'll just delete for now? it seems like commenting doesn't work from within the directive